backend/tools/rag_recipe_tool.py

```
@@ -0,0 +1,216 @@
+                model="text-embedding-3-small"
+            )
+
+            self.vectorstore = FAISS.load_local(
+                str(VECTORSTORE_PATH),
+                embeddings,
+                allow_dangerous_deserialization=True
+            )
+
+            print(f"[RAG] 벡터스토어 로드 완료: {VECTORSTORE_PATH}")
+
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)